car-service/src/main/kotlin/com/curious/programmers/carservice/CarController.kt

```diff
@@ -0,0 +1,29 @@
+package com.curious.programmers.carservice
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RestController
+import reactor.core.publisher.Flux
+
+
+data class AddCarRequest(val id: String, val make: String, val model: String)
+
+
+@RestController
+class CarController {
+
+    @Autowired
+    lateinit var carRepository: CarRepository
+
+
+    @PostMapping("cars")
+    fun post(@RequestBody car: AddCarRequest): Flux<CarAggregate> {
+
+        var subscribe = carRepository.save(CarAggregate(car.id, car.make, car.model)).subscribe()
+
+        return carRepository.findAll()
+    }
+
+
```

Comment:
As for me, there are too many new lines here, but maybe it's only me... :P

Reply:
I believe working on code after work trying to squeeze it somewhere in between life and no life sometimes ends with stuff like that ;)